### DISCUSSION TITLE
Add a "Windows 11 Start Menu Styler"

### DISCUSSION THREAD

**Discussion by firminunderscore** [valinet/ExplorerPatcher#3271] [Ideas]
I've seen the [Windows 11 Start Menu Styler](https://windhawk.net/mods/windows-11-start-menu-styler) and I think it would be a good idea to add it or a similar function to ExplorerPatcher. I actually like the Windows 10 Start Menu, but this could be a little extra.

**Comment by pyrates999** [comment]
Since it is already part of windhawk, I see no need for it to be added to EP.  I run windhawk along side EP and it works great.
